in_toto/models/common.py

```diff
@@ -102,6 +102,7 @@ def verify_signatures(self, keys_dict):
       except:
         raise SignatureVerificationError(
             "Signature key not found, key id is '{0}'".format(keyid))
-      if not keys.verify_signature(key, signature, self.payload):
+      if not securesystemslib.keys.verify_signature(
```

Comment:
This can potentially raise an exception. For example, `securesystems.exceptions.UnknownMethodError` is raised if `key` specifies an unsupported key type.  I guess this function assumes these arguments are valid, or verified elsewhere? (Perhaps assumptions of these sort should be stated in the docstring.)